tests/fixtures/database_connection_fixture.py

```diff
@@ -1,18 +1,8 @@
-import pytest
 import pyexasol
-from tests.utils.parameters import db_params
-import ssl
+import pytest
+from pytest_itde.config import TestConfig
 
 
 @pytest.fixture(scope="session")
-def pyexasol_connection() -> pyexasol.ExaConnection:
-    conn = pyexasol.connect(
-        dsn=db_params.address(),
-        user=db_params.user,
-        password=db_params.password,
-        encryption=True,
-        websocket_sslopt={
-            "cert_reqs": ssl.CERT_NONE,
-        }
-    )
-    return conn
+def pyexasol_connection(itde: TestConfig) -> pyexasol.ExaConnection:
+    return itde.ctrl_connection
```

Comment:
I think test should create their own connection based on the db settings.
I would be happy to discuss this one with you ;).